docs/resources/monitor.md

```diff
@@ -44,6 +44,8 @@ resource "datadog_monitor" "foo" {
 }
 ```
 
+**Note:** APM Trace Metrics are now available as distributions and your existing monitors have been migrated.  To learn more, or to see examples of how to update your terraform definitions to utilize the new distribution metrics, see the [migration guide](https://docs.datadoghq.com/tracing/guide/ddsketch_trace_metrics/)
```

Comment:
The `docs` folder is autogenerated based on both the `examples` folder, and the `Description` field in each resource. 

We'd either have to add this as a note to a description, or create a new template for the monitor doc that hardcodes this in. The template would be `templates/resources/<resource name>.md[.tmpl]`. More info on that here - https://github.com/hashicorp/terraform-plugin-docs#templates

Though it'd be nicer if we could include this as a description of one of the elements if thats possible 🙂

Reply:
It sounds like a new template would be the best bet here. We didn't actually change anything in the terraform elements themselves.

Reply:
I was thinking about this more. I'm thinking it may be a better idea to just append to the resource description so it appears at the top of the page:
Adding to the description here - https://github.com/DataDog/terraform-provider-datadog/blob/master/datadog/resource_datadog_monitor.go#L31
Something like: `\n\n<Your note>`

If that doesn't work out/isn't how we want it to display, then I can help make a new template. I'd just like "new template" to be our last option since it means we'd be maintaining that to make sure it was up to date with the doc autogeneration tool changes.

Reply:
^ tried this (thanks for the help again Nick! 

Added a comment with the preview https://github.com/DataDog/terraform-provider-datadog/pull/939#issuecomment-783552845